CHANGELOG.md

```diff
@@ -257,4 +257,4 @@ Changes since 0.9.2:
 
 This should be a sane polyfill for PHP 7's `random_bytes()` and `random_int()`.
 We hesitate to call it production ready until it has received sufficient third
-party review.
\ No newline at end of file
+party review.
```

Comment:
This was GitHub. It added a line break.
It's best practise for UNIX-style files to have one empty line at the end.